bagel/pheno_utils.py

```diff
@@ -237,6 +237,12 @@ def find_unused_missing_values(
     return all_unused_missing_vals
 
 
+def get_rows_with_empty_strings(df: pd.DataFrame, columns: list) -> list:
+    """Returns the ids of rows with empty strings"""
+    empty_row = df[columns].applymap(lambda cell: cell == "").apply(lambda row: any([value for value in row]), axis=1)
```

Comment:
This is a cool way to check for 'empty' cells! One thought - we're currently using `""` as our empty indicator due to our decision not to interpret `nan`s during the `read_csv` step / keeping everything as strings, right? I've read that `apply` / `applymap` can be time costly on large dataframes - what about replacing all the "" with `nan` first and then using `.isnull()` / `.isna()` on the dataframe to get the indices?

Reply:
Ah, that's an interesting point. I'm against applying this fix now because we haven't encountered a slowdown because of these `apply` statements, so let's see if this becomes an issue first. In addition, right now our bottleneck is almost certainly `pybids` with its sqlite initialization. But we could add a comment here with your suggestion in case things do slow down, so we remember it (or remind ourselves to check this sections as the slowdown culprit in a profile).